README.md

```diff
@@ -116,3 +120,74 @@ Modify `nrf802154_sniffer.py`:
     ```python
     #DLT='802.15.4'
     ```
+
+## Troubleshooting
+
+#### Sniffer capture hangs when another Wireshark process is started with nRF BLE sniffer extcap
+This issue is caused by BLE sniffer extcap, which discovers connected sniffers during Wireshark startup by actively sending data to all serial ports.
+Because Linux applications primarily use advisory locking, there is nothing stopping other applications from opening and writing data to the serial port, causing unexpected behaviour.
+Currently, the only way to avoid this issue is to disable the nRF BLE sniffer extcap by removing the executable flag:
+```bash
+sudo chmod -x <extcap_path>/nrf_sniffer.py
+```
+
+#### Sniffer capture hangs with `ModemManager` service enabled on Linux
+On some occasions the `ModemManager` may send AT commands to the sniffer. To avoid it do one of the following:
+* Disable `ModemManager` service
+```bash
+sudo systemctl stop ModemManager.service
+sudo systemctl disable ModemManager.service
+```
+* Create `udev` rule (if `ModemManager` runs on `DEFAULT` or `PARANOID` policy):
+    1. Create new file `/etc/udev/rules.d/99-mm-blacklist.rules` and write the below configuration:
```

Comment:
Create new file `/etc/udev/rules.d/99-mm-blacklist.rules` and include in it the following configuration:

Reply:
Fixed.